cli/client/errors.go

```diff
@@ -1,17 +1,9 @@
-package commands
+package client
 
 import (
-	"errors"
 	"fmt"
 )
 
-var (
-	errMissingAuthToken      = errors.New("missing AUTH_TOKEN")
-	errMissingLeaseToken     = errors.New("missing lease token")
-	errMissingServer         = errors.New("missing IP")
-	errMissingKubeConfigFile = errors.New("missing kubeconfig file")
-)
-
```

Comment:
fwiw, the reason I deleted these was that in the _only_ place(s) each of them was used, we were immediately doing a `log.Fatalf`. There isn't any sense returning a well-defined error when we always end execution before doing so (i.e. we don't).